client/ayon_deadline/plugins/publish/global/submit_publish_job.py

```diff
@@ -466,6 +467,7 @@ def process(self, instance):
         # Inject deadline url to instances to query DL for job id for overrides
         for inst in instances:
             inst["deadline"] = instance.data["deadline"]
+            inst["deadline"].pop("job_info")
```

Comment:
I got this error here:
```python
Traceback (most recent call last):
  File "C:\Users\Mustafa Taher\AppData\Local\Ynput\AYON\dependency_packages\ayon_2411052009_windows.zip\dependencies\pyblish\plugin.py", line 528, in __explicit_process
    runner(*args)
  File "E:\Ynput\ayon-deadline\client\ayon_deadline\plugins\publish\global\submit_publish_job.py", line 470, in process
    inst["deadline"].pop("job_info")
KeyError: 'job_info'
```

Reply:
Please provide what you were publishing from where with what configuration? So we can identify the point where `CollectJobInfo` did not run on the instance where it should have or where that data is lacking.

Reply:
After trying multiple times, the issue happens when publishing render with separated AOVs.
and it fails on the second iteration of the for loop.

Reply:
I think we may be cloning the instance data then without making deep copies of its data? OR - the runtime instances do not get a copy of the `job_info` data but they should have.

Reply:
yes, I think we should take a copy instead. 
maybe: 
```suggestion
            inst["deadline"] = instance.data["deadline"].copy()
            inst["deadline"].pop("job_info")
```

Reply:
Likely even a `copy.deepcopy` to be entirely explicit about this.